airflow/cli/cli_parser.py

```diff
@@ -468,7 +468,17 @@ def string_lower_type(val):
     help="Don't preserve purged records in an archive table.",
     action="store_true",
 )
-
+ARG_DB_EXPORT_FORMAT = Arg(
+    ("--export-format",),
+    help="The file format to export the cleaned data",
+    choices=("csv",),
+)
+ARG_DB_OUTPUT_PATH = Arg(
+    ("--output-path",),
+    metavar="FILEPATH",
+    help="The output to export the cleaned data",
+    required=True,
```

Comment:
Should we have a default directory instead of making this a required option?

Reply:
I think required makes sense